packages/client/src/forms/register/queries/registration.ts

```diff
@@ -17,7 +17,7 @@ import {
 } from '@opencrvs/gateway/src/graphql/schema.d'
 
 export const FETCH_REGISTRATION = gql`
-  query fetchRegistration($identifier: ID!) {
+  query queryRegistrationByIdentifier($identifier: ID!) {
```

Comment:
I had to add a name / rename almost all of our queries so the type generator is able to use unique names for the new types. The result type for this query looks like this

```ts
export type QueryRegistrationByIdentifierQuery = {
  __typename?: 'Query'
  queryRegistrationByIdentifier?: {
    __typename?: 'BirthRegistration'
    id: string
    child?: {
      __typename?: 'Person'
      id?: string | null
      birthDate?: string | null
      gender?: string | null
      name?: Array<{
        __typename?: 'HumanName'
        use?: string | null
        firstNames?: string | null
        familyName?: string | null
      } | null> | null
    } | null
  } | null
}
```